src/server/svr_jobfunc.c

```diff
@@ -3810,6 +3811,8 @@ eval_resvState(resc_resv *presv, enum resvState_discrim s, int relVal,
 				else
 					*psub = RESV_RUNNING;
 				*pstate = RESV_RUNNING;
+				if(!presv->ri_qs.ri_tactive)
```

Comment:
I think this will cause problem with standing reservation. First running occurrence of standing reservation will set this variable and then every subsequent occurrence will never log an accounting log.